file INSTALL cannot copy file liblcmtypes_bot2-core.a

https://drake-cdash.csail.mit.edu/viewBuildError.php?onlydeltap&buildid=55146

```
CMake Error at lcmtypes/cmake_install.cmake:117 (file):
  file INSTALL cannot copy file
  "/Users/elcapitan/workspace/mac-clang-continuous-release/pod-build/externals/bot_core_lcmtypes/lcmtypes/liblcmtypes_bot2-core.a"
  to
  "/Users/elcapitan/workspace/mac-clang-continuous-release/build/lib/liblcmtypes_bot2-core.a".
Call Stack (most recent call first):
  cmake_install.cmake:32 (include)


make[3]: *** [install] Error 1
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -211,13 +211,13 @@ else()
     CMAKE_ARGS -DBUILD_SHARED_LIBS=ON)
 endif()
 
-# bot_core_lcmtypes
-drake_add_external(bot_core_lcmtypes PUBLIC CMAKE
-  DEPENDS lcm)
-
 # libbot
 drake_add_external(libbot PUBLIC CMAKE
   DEPENDS lcm)
+
+# bot_core_lcmtypes
+drake_add_external(bot_core_lcmtypes PUBLIC CMAKE
+  DEPENDS lcm libbot) # Conflicts with libbot; ensure this is built after
 
 # director
 drake_add_external(director PUBLIC

